script/metric/step_5_import_metric_value.rb

```diff
@@ -0,0 +1,108 @@
+
+require File.expand_path('../../../config/environment',  __FILE__)
+require 'colorize'
+
+if ARGV.length == 0
+  puts "Please include a file path".red
+  puts "EX: ruby script/import_metric_value.rb "\
+       "script/metric/csv/metric_to_import.csv".green
+  exit
+end
+
+Card::Env[:protocol] = "http://"
+Card::Env[:host] = "http://wikirate.org"
+Card::Auth.current_id = Card.fetch_id "Richard Mills"
+Card::Auth.as_bot do
+  source_cache = Hash.new
+  csv_file_path = ARGV[0]
+  puts "CSV file to import : #{csv_file_path}"
+  CSV.foreach(csv_file_path, encoding: 'windows-1251:utf-8', headers: true,
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.